.github/workflows/continuous-integration.yml

```diff
@@ -17,53 +17,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, macos-13, macos-latest, windows-2022]
-        gfortran-version: [12, 13, 14]
-        gfortran-from: [system, conda]
+        rte-kernels: [default, accel]
         fpmodel: [DP, SP]
+        build-type: [Debug, RelWithDebugInfo]
+        # Add Release config?
+        # Exclude RelWithDebugInfo/accel combinations?
         exclude:
-          - os: ubuntu-24.04
-            gfortran-from: conda
-          - os: macos-13
-            gfortran-from: system
-          - os: macos-13
-            gfortran-version: 14
-            gfortran-from: conda
-          - os: macos-latest
-            gfortran-from: system
-          - os: macos-latest
-            gfortran-version: 14
-            gfortran-from: conda
-          - os: windows-2022
+        - build-type: RelWithDebugInfo
+          rte-kernels: accel
```

Comment:
I don't see why we should not run this case.

Reply:
I thought to exclude this because it would never be used. Builds for use on CPU architectures will bundle the default kernels. I did want to test the `accel` kernels with max debugging in case we introduce coding errors.